ledger-bridge.js

```diff
@@ -47,6 +47,8 @@ export default class LedgerBridge {
                            this.updateTransportTypePreference(replyAction, 'u2f')
                         }
                         break
+                    case 'ledger-make-app':
+                        this.attemptMakeApp(replyAction);
```

Comment:
This requires a `break` afterward.

Reply:
done in https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/125/commits/ed2be6b3e6e646ec866655cad59509c261b0ad8e